lib/cloudflare/accounts.rb

```diff
@@ -24,9 +24,19 @@
 
 require_relative 'representation'
 require_relative 'paginate'
+require_relative 'kv/namespaces'
 
 module Cloudflare
 	class Account < Representation
+
+		def id
+			value[:id]
+		end
+
+		def kv_namespaces
```

Comment:
Do you think maybe `stores` or `storage` could be a more generic name?

Reply:
I was going with what cloudflare calls them in their docs. I thought about being more generic but since the url has "storage" in it, I thought it might be better to have it be more specific in case they added other storage types. I'm open to whatever you prefer though.

Reply:
The only reason why I don't like `kv` is because it's an abbreviation and abbreviations are not very accessible (e.g. visually impaired programmers, non-native speakers). When I look at the name `kv_namespaces`, I sort of think, is it a list of databases? Could there be something like `sql_namespaces`, why is `kv` significant? What does `namespaces` tell me?

You are in a better position to figure out how it should be named. One way to help this is to figure out how you'd use it in code and what makes it more readable for users, e.g.

```ruby
things = account.kv_namespaces[:things]

things["key"] = value
```

Reply:
I think kv_namespaces is understandable. All through the interface/documentation they refer to it as either Workers KV or just KV (i.e. KV Bandwidth, KV Requests). Here's a link to their documentation: https://api.cloudflare.com/#workers-kv-namespace-properties 

A sample string for this would be:

```
things = account.kv_namespaces.find_by_title('things')
things.set_value('xyz', 1)
things.get_value('xyz')
```

I could potentially see turning it into `account.kv.namespaces` but that seemed like overkill since nothing else lives under `kv`, it just means an extra method call every time. I also considered `account.storage.kv_namespaces` or `account.storage.kv.namespaces` but, like in my last example, it seemed like extra method calls for little benefit.

Reply:
Are there any other interfaces under `account/storage` URI? or just all this `kv` stuff?

Reply:
From what I can find, this is the only thing currently under `/storage`.

Reply:
If I had to choose, I'd probably be leaning like slightly towards using something simpler than `kv_namespaces` but only slightly. I tend to find names that reflect what they provide to the user, not how they are implemented/what they do. But, I really respect your effort here so you choose whatever you think is good.

Reply:
I would like to come up with something better, but I'm having a hard time coming up with anything that can be more generic while still being flexible enough that we could add additional endpoints to it if something came up. If you have a suggestion, I'll go with it, otherwise, I think we should just stick with this for now.

Reply:
When I review the full set of changes before making a release, I will think about it in more detail, once I have the full scope of these changes in my working memory. Let's just leave it as is for now.

Reply:
Sounds good